python/paddle/v2/fluid/__init__.py

```diff
@@ -34,13 +34,14 @@
 from core import LoDTensor, CPUPlace, CUDAPlace
 from distribute_transpiler import DistributeTranspiler
 from distribute_transpiler_simple import SimpleDistributeTranspiler
+from concurrency import *
```

Comment:
According to the [PEP8 style](https://www.python.org/dev/peps/pep-0008/#id23):

> Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools.

Reply:
Thank you for pointing that out. Fixed in the latest commit.